### PR TITLE
fix: Data Analysis with Python- Incorrect answer for Numpy Operations

### LINKED ISSUE #38966: Data Analysis with Python Course - Numpy Operations - incorrect answer
**Describe your problem and how to reproduce it:**
None of the available answers are correct. Without the `5` second option (marked as solution) would have the right numbers.
>What is the value of `a` after you run the following code?:
```
a = np.arange(5)
a + 20
```
>Answers
```
[20, 21, 22, 24, 24]
[0, 1, 2, 3, 4, 5]
[25, 26, 27, 28, 29]
```

**Add a Link to the page with the problem:**
https://www.freecodecamp.dev/learn/data-analysis-with-python/data-analysis-with-python-course/numpy-operations

### PR DESCRIPTION
**Pull request by niteshseram**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #38966 

Fix the incorrect option 2 and the correct answer is changed from option 2 to 1

### CODE REVIEW

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/numpy-operations.english.md [hunk 1]
```diff
@@ -24,11 +26,10 @@ question:
...
   answers:
-    - "[20, 21, 22, 24, 24]"
-    - "[0, 1, 2, 3, 4, 5]"
-    - "[25, 26, 27, 28, 29]"
-  solution: 2
-```
+    - '[20, 21, 22, 24, 24]'
+    - '[0, 1, 2, 3, 4]'
+    - '[25, 26, 27, 28, 29]'
+  solution: 1
```
Comment (moT01) [inline curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/numpy-operations.english.md:32]:
```suggestion
  solution: 2
```

**Review by moT01 — CHANGES_REQUESTED** [review]
Note that this is not zero based indexing. I think 2 is the solution.

**Comment by niteshseram** [comment]
I have commit the changes with option 2.
Though when I tried running the code, the output was option 1

**Comment by moT01** [comment]
Can you show me? I see option 2...
Screen Shot 2020-05-31 at 9 52 14 AM

**Comment by niteshseram** [comment]
@moT01 Sorry,that was my bad. You are right. I was printing a+20 directly thats the reason I got option 1. I just realized a+20 is not store in any variable.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Comment by niteshseram** [comment]
Thanks and this was my first open source contribution

**Comment by moT01** [comment]
Congratulations @niteshseram 🎉 Thanks for contributing to freeCodeCamp 🎉 Once another person takes a look at this and says it looks good, we can merge it.

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Pull request merged**